### ISSUE TITLE
No Meteorites Worldgen?

### ISSUE THREAD

**Issue by Xiaminou** [thiakil/Applied-Energistics-2#2]
#### Description
The compass is spinning like there are no meteorites even after exploring 1000² blocks. Is meteorite worldgen not in 1.11.2 yet?

Minecraft Version: 1.11.2
AE2 Version: rv5-alpha 0
Forge Version: 2388

**Comment by thiakil** [comment]
I have yet to test it, so likely broken

**Comment by thiakil** [comment]
May have actually been fixed after I fixed some mappings problems, will test tonight

**Comment by thiakil** [comment]
Please test with alpha-1. Worked in my quick tests (though they were all underground)

https://github.com/thiakil/Applied-Energistics-2/releases/tag/rv5-alpha-1

**Comment by Xiaminou** [comment]
Well, I found some meteorites today after updating to alpha-1, but I also noticed the compass always spins like there are no meteorite when it's in your hotbar (I haven't played for a while but I remember it pointing to the meteorite evenwhen it's just in your hotbar) but not in your hand, so I may have just been very unlucky or even missed some. Or meteorite worldgen is set to retrogen by default, I guess we'll never know. But anyway problem solved, thank you.
